Makefile

```diff
@@ -1,9 +1,10 @@
-version?=10-internal
+version?=internal
 package?=MarkLogic.rpm
-REPONAME=marklogic-centos
+REPONAME=local-dev
+docker_regestry?=ml-docker-db-dev-tierpoint.bed-artifactory.bedford.progress.com
 repoDir=marklogic
 docker_build_options=--compress
-test_image?=ml-docker-dev.marklogic.com/${repoDir}/marklogic-server-centos:${version}
+test_image?=${docker_regestry}/${repoDir}/marklogic-server-centos:${version}
```

Comment:
I guess there is a typo here:
regestry = registry ?

Reply:
Fixed.